docs/MIGRATION-1.8-to-1.9.rst

```diff
@@ -0,0 +1,205 @@
+   `ODC Enhancement Proposal 8 <https://github.com/opendatacube/datacube-core/wiki/ODC-EP-008>`_
+
+   The handling of lineage in the legacy postgres index driver has not changed - the postgres driver does NOT support
+   the new lineage API.
+
+6. Support for multi-dimensional loading of hyperspectral datasets (Coming Soon)
+
+   This is a work in progress and will not be available in 1.9.0. It will appear in a later 1.9.x release.
+
+7. The long-deprecated "ingestion" workflow and "excecutor" API have both been removed.
+
+8. Multiple locations per dataset is now deprecated.
+
+The New Postgis Index Driver
+----------------------------
+
+Configuration
++++++++++++++
+
+The configuration for a postgis index looks the same as the configuration for a legacy postgres index, you simply
+set the `index_driver` setting to `postgis`::
+
+   [default]
+      alias: prod
+
+   [old]
+      index_driver: postgres
+      db_hostname: production.dbs.internal
+      db_database: odc_prod
+      db_username: odc
+      db_password: secret_and_secure
+
+   [new]
+      index_driver: postgis
+      db_hostname: dev.dbs.example.net
+      db_database: odc_dev
+      db_username: odc
+
+Initialisation
+++++++++++++++
+
+You then initialise the database as previously, using `system init` command (-E new says to use the `new` environment
+from the configuration file)::
+
+   datacube -E new system init
+
+You should also create Postgis spatial indexes for any CRS you want to be able to search on (note that an EPSG:4326
+spatial index is created by default).   Postgis spatial indexes should be created before indexing any data where
+possible.  Adding a new spatial index to a populated index will be very slow.  For example to create a spatial index
+for EPSG:3577::
+
+   datacube -E new spindex create 3577
+
+Migrating (Cloning) Data From a Postgres Index
+++++++++++++++++++++++++++++++++++++++++++++++
+
+To clone data from an old index to a new one (the two indexes may use different index drivers)::
+
+   datacube -E new system clone old
+
+Note that the target index is specified with the `-E` flag and the source index is provided as an argument to the
+`system clone` command.
+
+Data that cannot be directly copied is skipped, e.g.:
+
+* Non-EO3 compatible data cannot be copied from a `postgres` index into a `postgis` index.
+* External lineage information cannot be copied from a `postgis` index to a `postgres` index.
+
+The clone command supports the following options:
+
+* `--skip-lineage` If set, lineage data is not copied.  Default is to NOT skip lineage (to attempt to copy lineage data)
+* `--lineage-only`  If set, ONLY lineage data is copied.
+* `--batch-size N`  Index cloning is batched for performance. This option specifies the number of records to write to
+  the target database at a time.  Default is 1000.
+
+Geospatial search
++++++++++++++++++
+
+Geopolygons for spatial search can be passed to `dc.load()`::
+
+   dc.load(...., geopolygon=poly, ...)
```

Comment:
Searching for data via the `geopolygon` param is already possible in version 1.8. What has changed in 1.9? We should make any changes/improvement in functionality obvious here